spec/app_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './app'
+
+describe Chitter do
+  feature 'Test' do
+    scenario "stores a list of test cheets" do
+      visit('/')
+
+      expect(page).to have_content('Test Cheet One')
```

Comment:
Related to the comment about the location of test code, if I read this test in isolation I have no idea where Test Cheet One etc come from. It's much clearer to another dev if the test setup is located in the same place as the test, as then they can see precisely what's going on.